resources/js/components/user/Add/AddUser.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <v-btn color="primary" @click="onClickAddButton()">
+        Add User
+    </v-btn>
+</template>
+
+<script>
+export default {
+    name: "AddUser",
+    data() {
+        return {};
+    },
+    methods: {
+        onClickAddButton() {
+            this.$router.push("/add");
+        }
+    }
+};
+</script>
```

Comment:
If the component has only one button and one function => please don't create the child component because on the project source code, the number of files will be very large.
Besides, you can not need to use function to switch router. You can user `<router-name></router-name>` tag as a button.